openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -154,13 +157,20 @@ def set_logged_in_cookies(request, response, user):
     return response
 
 
-def refresh_jwt_cookies(request, response, user):
+def refresh_jwt_cookies(request, refresh_status, user):
     """
     Resets the JWT related cookies in the response for the given user.
     """
     cookie_settings = standard_cookie_settings(request)
+    response = JsonResponse({})
     _create_and_set_jwt_cookies(response, request, cookie_settings, user=user)
-
+    response.content = json.dumps(
+        {
+            'success': refresh_status,
```

Comment:
Why not drop `refresh_status` and keep this hard-coded to True as it was?  You could add to the docstring that any failure will raise an exception.